docs/docs/guides/reverse_proxy.md

```diff
@@ -3,25 +3,38 @@ id: reverse_proxy
 title: Setting up a reverse proxy
 ---
 
-This guide outlines the basic configuration steps needed to expose your Frigate UI to the internet.
-A common way of accomplishing this is to use a reverse proxy webserver between your router and your Frigate instance.
-A reverse proxy accepts HTTP requests from the public internet and redirects them transparently to internal webserver(s) on your network.
-
-The suggested steps are:
-
-- **Configure** a 'proxy' HTTP webserver (such as [Apache2](https://httpd.apache.org/docs/current/) or [NPM](https://github.com/NginxProxyManager/nginx-proxy-manager)) and only expose ports 80/443 from this webserver to the internet
-- **Encrypt** content from the proxy webserver by installing SSL (such as with [Let's Encrypt](https://letsencrypt.org/)). Note that SSL is then not required on your Frigate webserver as the proxy encrypts all requests for you
-- **Restrict** access to your Frigate instance at the proxy using, for example, password authentication
+This guide outlines the basic configuration steps needed to set up a reverse proxy in front of your Frigate instance.
+
+A reverse proxy is typically needed if you want to set up Frigate on a custom URL, on a subdomain or on a host serving multiple sites. It could also be used to set up your own authentication provider or for more advanced HTTP routing.
```

Comment:
```suggestion
A reverse proxy is typically needed if you want to set up Frigate on a custom URL, on a subdomain, or on a host serving multiple sites. It could also be used to set up your own authentication provider or for more advanced HTTP routing.
```